### PR TITLE
Feature/gen readme

### PR DESCRIPTION
**Pull request by PaigeCD**
# Add ability to generate README

## Description

Add the readme_gen function which enable the use of the --gen-readme flag. As per the user story, this README file contains simple instructions on running GatorGradle.

### Type of Change

- [x] Feature
- [ ] Bug fix
- [ ] Documentation

## Contributors

- @PaigeCD 

## Reminder

All GitHub Actions should be in a passing state before any pull request is merged.

### CODE REVIEW

**Comment by PaigeCD** [comment]
Screenshot from 2022-02-03 12-10-19
This is what the generated README file will look like. Please note that the title and intro sentence will reflect the assignment's default name. It is also worth noting that the generated file will be called `README.md` and the name of this example was purely for testing purposes.

**Review by WesleyL30 — APPROVED** [review]
Looks good!

File: gatorconfig/gator_config.py [hunk 1]
```diff
@@ -73,6 +71,27 @@ def cli_input(
         print("\nUse \"--overwrite\" to rewrite this file.")
     #print(files)
     actions_configuration.create_configuration_file('.github/workflows/grade.yml')
+    readme_gen(gen_readme, output_path)
+
+
+def readme_gen(gen_readme: bool, output_path: Path):
+    """Generate basic README in current directory."""
+    if gen_readme is True:
```
Comment (ColemanKobe) [inline gatorconfig/gator_config.py:79]: My pet peeve is just that you could just say `if gen_readme:`. However, it doesn't really matter

**Review by ColemanKobe — APPROVED** [review]
lgtm

**Pull request merged**